wifiphisher/common/accesspoint.py

```diff
@@ -5,17 +5,35 @@
 import os
 import time
 import subprocess
+from subprocess import check_output
+import threading
 import wifiphisher.common.constants as constants
 import roguehostapd.config.hostapdconfig as hostapdconfig
 import roguehostapd.apctrl as apctrl
+import wifiphisher.common.victim as victim
 
 
 class AccessPoint(object):
     """This class forks the softAP."""
 
+    # Instance will be stored here.
+    __instance = None
+
+    @staticmethod
+    def get_instance():
+        if AccessPoint.__instance is None:
```

Comment:
You can simplify this to
```
if AccessPoint.__instance:
    return AccessPoint.__instance
else:
    AccessPoint()
```

Reply:
hmm actually we need to return at the else too and i dont think it would look as good...